mw-webapp/src/logic/allWaysPage/AllWaysPage.tsx

```diff
@@ -50,16 +76,28 @@ export const AllWaysPage = () => {
 
   return (
     <>
```

Comment:
extra wrapper

Reply:
It's not extra wrapper, there are multiple children